src/resources/OrganizationAccess/AccessInterfaces.ts

```diff
@@ -0,0 +1,5 @@
+export interface AccessModel {
+    accessLevel: string[];
```

Comment:
no jsdoc? 👀

Reply:
hmm I wasn't sure exactly what these are for 🙈 - I guess I could try to provide what swagger API call has and add them here 👍

Reply:
yeah likes those descriptions 👀 
<img width="1149" alt="image" src="https://user-images.githubusercontent.com/63734941/192623473-ca2e25b5-3395-4069-8120-2387fc6199aa.png">

Reply:
easy copy pasta 🍝